Gemfile

```diff
@@ -3,6 +3,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.2.1'
 
+gem 'bullet'
+
```

Comment:
- [ ] As required for the project, kindly ensure that you don't to [use the bullet gem](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Do%20not%20use,after%20the%20fix) and to add screenshots of the logs before and after the fix to the PR description. Right now you only added the screenshots of the logs **After** the fix,  ensure to also ad **Before** the fix